modules/site-health/audit-database/class-perflabdbmetrics.php

```diff
@@ -0,0 +1,433 @@
+<?php
+/**
+ * Performance Lab database metrics computation module
+ *
+ * @package performance-lab
+ * @group audit-database
+ *
+ * @since 1.4.0
```

Comment:
Please use throughout the PR:
```suggestion
 * @since n.e.x.t
```

There should be no specific `@since` tag in new code.